packages/catalog-client/api-report.md

```diff
@@ -43,6 +43,10 @@ export interface CatalogApi {
     name: EntityName,
     options?: CatalogRequestOptions,
   ): Promise<Entity | undefined>;
+  getEntityFacets(
+    request: GetEntityFacetsRequest,
+    options?: CatalogRequestOptions,
+  ): Promise<GetEntityFacetsResponse>;
```

Comment:
This is a breaking change that needs highlighting. Again thinking if there are some other options that make more sense here. These large interfaces really add a lot of friction when they show up in the core parts

Reply:
Updated changeset!